src/babashka/neil/new.clj

```diff
@@ -13,6 +13,13 @@
   [template]
   (contains? (set (map name (keys (ns-publics 'org.corfield.new)))) template))
 
+(defn- built-in-templates []
+  (require 'org.corfield.new)
+  (let [blacklist #{'create}]
```

Comment:
The word `blacklist` can trigger some bad connotations with some people, let's use the word `non-templates`